app/src/main/java/uk/ac/ebi/atlas/experimentpage/json/JsonDifferentialExperimentController.java

```diff
@@ -125,4 +125,17 @@ public String differentialRnaSeqExperimentData(
                 (DifferentialExperiment) experimentTrader.getExperiment(experimentAccession, accessKey),
                 accessKey, preferences));
     }
+
+    @RequestMapping(value = "/json/experiments/{experimentAccession}",
+            produces = "application/json;charset=UTF-8",
+            params = "type=PROTEOMICS_DIFFERENTIAL")
+    @ResponseBody
+    public String differentialProteomicsExperimentData(
+            @ModelAttribute("preferences") @Valid MicroarrayRequestPreferences preferences,
```

Comment:
Why `MicroArrayRequestPreferences`? Please use a suitable class, create it if necessary.